src/lender/test/memberlist.t.sol

```diff
@@ -24,21 +24,38 @@ contract MemberlistTest is Math, DSTest {
 
     uint memberlistValidity = safeAdd(now, 8 days);
     Memberlist memberlist;
+    Memberlist testMemberlist;
```

Comment:
I think this is a bit hacky, makes it hard to read. Better to just use `address user1;`

Reply:
You can just remove them